2018/ffs-wp.md

```diff
@@ -0,0 +1,277 @@
+für "Open-Source-Software" auf:
+
+    3.1 Dual-licensing
+    3.2 Selling professional services
+    3.3 Selling of branded merchandise
+    3.4 Selling of certificates and trademark use
+    3.5 Selling software as a service
+    3.6 Partnership with funding organizations
+    3.7 Voluntary donations
+    3.8 Bounty driven development
+    3.9 Pre-order/crowdfunding/reverse-bounty model
+    3.10 Crowdsourcing
+    3.11 Advertising-supported software
+    3.12 Selling of optional proprietary extensions
+    3.13 Selling of required proprietary parts of a software product
+    3.14 Selling of proprietary update systems
+    3.15 Re-licensing under a proprietary license
+    3.16 Obfuscation of source code
+    3.17 Delayed open-sourcing
+    3.18 Open sourcing on end-of-life
+
+
+ Für die Projekte mit der Zielgruppe der Endbenutzer\*innen kommen bestimmte Modelle
+ nicht in Frage bzw. sind nicht wünschenswert (Unterschied zwischen Open-Source und freier Software).
+ Im Folgenden sind Ausschlussgründe und die davon betroffenen Geschäftsmodelle aufgelistet.
+
+
+- Nicht sinnvoll, Weil kein Weiterverkauf angestrebt wird
+    - Dual-licensing
+    - Re-licensing under a proprietary license
+- Zielkonflikt: Perfekte Software braucht keinen Support/Schulung
+    - Selling professional services
+- Problem der Datensouveränität.
+    - Selling software as a service
+
+
+- Widerspricht dem Ziel maximalen FLOSS-Anteils
+    - Selling of optional proprietary extensions
+    - Selling of required proprietary parts of a software product
+    - Selling of proprietary update systems
+    - Obfuscation of source code
+    - (Delayed open-sourcing)
+
+- Nicht nachhaltig
+    - Open sourcing on end-of-life
+
+Es bleiben also folgende Modelle übrig:
+
+    3.3 Selling of branded merchandise
+    3.4 Selling of certificates and trademark use
+    3.6 Partnership with funding organizations
+    3.7 Voluntary donations
+    3.8 Bounty driven development
+    3.9 Pre-order/crowdfunding/reverse-bounty model
+    3.10 Crowdsourcing
+    3.11 Advertising-supported software
+
+## Ideenskizze zur Kombination (einiger) dieser Methoden
+
+### Vorbemerkung
+"Voluntary donations" klingt, als wäre die Zahlung "gegen die ökonomische Vernunft" und allein
+aus irrationalem Altruismus oder Mitleid. Ohne tiefer in die Kritik des Modells des Homo-Oeconomicus
+einzusteigen, zwei Anmerkungen:
+
+- In vielen anderen Bereichen sind Menschen aber bereit, erhebliche Mittel "entgegen der ökonomischen Vernunft" auszugeben, Stichwort: Statuskonsum.
+- Wie oben schon erwähnt, kann die finanzielle Unterstützung eines Projektes von dem man individuell profitiert auch mit informiertem Eigeninteresse begründet werden.
+
+
+### Kombinationsmodell
+Es gibt eine Organisation (Arbeitstitel: "Funding Freedom Society" (FFS)).
+Bei dieser sind alle an Finananzierung interessierten FLOSS-Projekte registriert und haben ein "Konto".
```

Comment:
Kritik: typisches Plattform-Geshcäftsmodell mit enormen Gebühren und starker Abhängigkeit von der selbst proprietären plattform.

Reply:
Liberapay z.B. finanziert sich wiederum durch Spenden der User. Das ist fair. Sowas wäre ja auch eine Möglichkeit.

Reply:
"enorme Gebühren": Ich hatte erstmal an 10% gedacht, wobei das eher ins blaue geraten war. Vielleicht ist das zu hoch. Das Librepay-Modell wäre auch eine Variante (erfordert aber ggf. mehr Überzeugungsarbeit.)

Reply:
Ich finde es immer gut, wenn es *alternative Möglichkeiten* gibt, 3 Level: Eilig, Geizig, Klandestin. Manche Menschen sind geizig und wollen keinem Mittelsmann was gönnen; für diese sollte eine weitgehend direkte Möglichkeit vorhanden sein. Andere wiederum wünschen es sich "one-click" ohne Aufwand, dann eben integriert mit dem "Store" oder den üblichen (nicht nur den politisch besseren aber dafür weniger bekannten) Mittelsmännern die ggf. ihren nicht gerade kleinen Anteil abgreifen. Eine absolute Minderheit macht sich Gedanken über soetwas wie Anonymität; aber auch das ist in Zeiten von Kryptowährungen kein Problem. Bsp.: https://netzpolitik.org/spenden/?via=nav bzw. diverse Vereine: https://c3d2.de/unterstuetzen.html . Patreon offenbart einen weiteren Fall: Exklusivität, es gibt für die Unterstützer (exklusiv oder vor Veröffentlichung für alle) Zugriff auf Content. Das ist schon nahe am Merchandise wo es ab einer „Spende“ >x ein Stickerpaket etc. gegen Bekanntgabe der Versandadresse gibt.